core/src/avm2/globals/flash/display/StageAspectRatio.as

```diff
@@ -0,0 +1,13 @@
+package flash.display
+{
+    [API("668")]
+    public final class StageAspectRatio
+    {
+        [API("681")]
+        public static const ANY:String = "any"
+        [API("668")]
+        public static const LANDSCAPE:String = "landscape"
+        [API("668")]
+        public static const PORTRAIT:String = "portrait"
```

Comment:
You are missing semicolons, hence the errors. Also, I think annotating these constants as `[API("668")]` is unnecessary, as the whole class is annotated that way.

Reply:
Fixed